build.gradle

```diff
@@ -427,11 +426,6 @@ dependencies {
     testRuntimeOnly "org.apache.kafka:kafka-metadata:${kafka_version}"
     testRuntimeOnly "org.apache.kafka:kafka-storage:${kafka_version}"
 
-
-
-    implementation "com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}"
```

Comment:
According to the [comment](https://github.com/opensearch-project/OpenSearch/issues/5526#issuecomment-1346906714), we can make this to `compileOnly`